### PR TITLE
Initial commit for AWS DynamoDb v.0.01

### PR DESCRIPTION
**Pull request by nikkusib**

### CODE REVIEW

**Comment by StarpTech** [comment]
Hi @nikkusib thanks for the contribution. I plan to move all technology based plugins in seperate repositorys to support CI. In a monorepo it is very hard to support CI. Therefore could you update the plugin to the newest hemera version? The plugin defintion was changed a bit.

**Comment by StarpTech** [comment]
Here you can see some examples 

- https://github.com/hemerajs/hemera-mongo-store
- https://github.com/hemerajs/hemera-arango-store

Don't forget to add your plugin on the readme list ;)

**Comment by StarpTech** [comment]
@nikkusib I added you as the contributor in https://github.com/hemerajs/hemera-dynamodb-store

**Comment by nikkusib** [comment]
Hi @StarpTech . First of all, thank you for the explanation. I will move my whole code to the new repository. BTW, are there any other further steps that I will need to do ? Thanks

**Comment by StarpTech** [comment]
1. Update your plugin to hemera 3.0
2. Change the plugin registration https://github.com/hemerajs/hemera-mongo-store/blob/master/index.js#L186
3. You can align you to the existing packages refer to linting, formating tools https://github.com/hemerajs/hemera-mongo-store

File: packages/hemera-dynamo-store/index.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    const store = new DynamoStore(db)
+    store.removeById(req, cb)
+  })
+
+  hemera.add({
+    topic,
+    cmd: 'updateById',
+    id: Joi.required(),
+    collection: Joi.string().required(),
+    UpdateExpression: Joi.string().required,
```
Comment (StarpTech) [inline packages/hemera-dynamo-store/index.js]: Please wrap it in a seperate property like `options`. The reason why is that every primitive value is relevant for pattern matching. A good practice is to wrap options in an object to exclude it.

**Comment by StarpTech** [comment]
Due to the new PR I close it.

**Pull request closed without merging**